lib/ansible/plugins/cliconf/eric_eccli.py

```diff
@@ -0,0 +1,98 @@
+#
+# Copyright (c) 2019 Ericsson AB.
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+---
+author: Ericsson IPOS OAM team
+cliconf: eccli
+short_description: Use eccli cliconf to run command on Ericsson ECCLI platform
+description:
+  - This eccli plugin provides low level abstraction apis for
+    sending and receiving CLI commands from Ericsson ECCLI network devices.
+version_added: "2.9"
+"""
+
+
+from ansible.module_utils.common._collections_compat import Mapping
+import re
+import time
+import json
+
+from itertools import chain
+
+from ansible.errors import AnsibleConnectionFailure
+from ansible.module_utils._text import to_text
+from ansible.module_utils.six import iteritems
+from ansible.module_utils.network.common.config import NetworkConfig, dumps
+from ansible.module_utils.network.common.utils import to_list
+from ansible.plugins.cliconf import CliconfBase, enable_mode
+
+
+class Cliconf(CliconfBase):
+
+    def get_config(self, source='running', flags=None, format=None):
+        return
+
+    def edit_config(self, candidate=None, commit=True, replace=None, comment=None):
+        return
+
+    def get(self, command=None, prompt=None, answer=None, sendonly=False, output=None, check_all=False):
+        if not command:
+            raise ValueError('must provide value of command to execute')
+        if output:
```

Comment:
We don't require the `output` parameter to be present in `get()`, so you can just remove the option if there is not intent to support it.